app/src/main/java/org/sil/gatherwords/MainActivity.java

```diff
@@ -36,10 +47,22 @@ public void onCreateSessionClick(View v) {
 
     // TODO: Switch to CursorAdapter
     private class SessionListAdapter extends BaseAdapter {
+        private List<Session> sessions;
+        DatabaseAccess databaseAccess;
+
+        SessionListAdapter() {
+            try {
+                databaseAccess = new DatabaseAccess(ad);
+                sessions = (List<Session>) databaseAccess.select("session").get();
+
+            } catch (InterruptedException | ExecutionException e) {
+                e.printStackTrace();
```

Comment:
Please do not use stdio.  If you want to log use the `Log` class; it allows you to differentiate (and filter) what level of error the message is.